functions.php

```diff
@@ -215,15 +218,17 @@ function add_section_markup_before( $content, $section ) {
 		$style_attrs .= 'color: '. $text_color_custom .'; ';
 	}
 
+	$title = ! empty( $title ) ? ' data-section-link-title="' . $title . '"' : '';
+
 	ob_start();
 ?>
-	<section class="<?php echo $section_classes; ?>" style="<?php echo $style_attrs; ?>">
+	<section id="<?php echo $section->post_name; ?>" class="<?php echo $section_classes; ?>" style="<?php echo $style_attrs; ?>"<?php echo $title; ?>>
```

Comment:
Actually, I'm going to refactor this for a couple of reasons. One, I think we can have a simpler way of doing the animate, and it should probably be done in the theme, so I'm going to refactor out the animation logic in the section-menu javascript.

Two, I think I'm going to go ahead and make id something that has to be set. This way, the id is independent from the post_name and post_title.